macroid-core/src/main/scala/macroid/Contexts.scala

```diff
@@ -45,12 +46,19 @@ object ContextWrapper {
 
   def apply[F](fragment: F)(implicit fragmentImpl: Fragment[F]): ActivityContextWrapper =
     ContextWrapper(fragmentImpl.activity(fragment))
+
+  def apply(view: View): ContextWrapper = view.getContext match {
+    case activity: Activity => ContextWrapper(activity)
+    case service: Service => ContextWrapper(service)
+    case app: Application => ContextWrapper(app)

```

Comment:
In this moment, I don't know when a `Context` in a `View` can be a ContextThemeWrapper, I always use views in activities, services or fragments

Anyway, I think that `DefaultContextWrapper` is a better option for this